website_multi_theme/models/ir_model_data.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+from odoo import models, api
+
+
+_logger = logging.getLogger(__name__)
+
+
+class IrModelData(models.Model):
+    _inherit = 'ir.model.data'
+
+    @api.model
+    def create(self, vals):
+        res = super(IrModelData, self).create(vals)
+        if vals.get('model') == 'ir.ui.view' \
+           and not self.env.context.get('duplicate_view_for_website'):
+            view = self.env['ir.ui.view'].browse(vals.get('res_id'))
+            if view.customize_show:
+                # Reload theme to make new customize_show available right after
```

Comment:
Put this explanation in the method docstring instead of here, as it's the purpose of the override.